tests/test_data.py

```diff
@@ -4,16 +4,39 @@
 from pathlib import Path
 import os
 import pandas as pd
-import cProfile
 import json
 from evaltools.data import (
     cvap, acs5, census, variables, submissions, tabularized, AssignmentCompressor,
-    remap
+    remap, estimatecvap
 )
 import us
+import geopandas as gpd
+import math
 
 root = Path(os.getcwd()) / Path("tests/test-resources/")
 
+def test_estimate_cvap():
```

Comment:
Added a test where I grabbed two 2010 tracts geographically distant from each other, matched them to 2020 block groups, and hand-calculated the correct values. The tests check whether the results in the dataframe match those expected results!